gr-qtgui/grc/qtgui_range.block.yml

```diff
@@ -53,19 +53,29 @@ parameters:
     dtype: int
     default: '200'
     hide: part
+-   id: outputmsgname
+    label: Message Property Name
+    dtype: string
+    default: 'value'
+    hide: 'part'
 -   id: gui_hint
     label: GUI Hint
     dtype: gui_hint
     hide: part
 value: ${ value }
 
+outputs:
+-   domain: message
+    id: value
```

Comment:
I'd call the output port out not value, the port name is independent from the message key.